docs/AboutUs.adoc

```diff
@@ -4,27 +4,28 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-The dummy content given below serves as a placeholder to be used by future forks of the project. +
+Project LISA was developed by the https://github.com/CS2103AUG2017-T13-B1[CS2103AUG2017-T13-B1] team. +
 {empty} +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
 === John Doe
 image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+{empty} [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+
+Role: Developer +
+Responsibilities:
 
-Role: Project Advisor
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Oscar Wang
```

Comment:
Please include your full name as per matric card, we will use this page for record keeping